diagnostic.md

```diff
@@ -7,40 +7,40 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+Clicked the fork button in the git UI. copied the URL: git@github.com:joehsuLM/git-diagnostic.git. On my local device, 'cd' to ~/wdi/diagnostics/ and use the command: `git clone git@github.com:joehsuLM/git-diagnostic.git`
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+cd into the newly cloned directory, 'git-diagnostic'
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+this can be done with a one line command: `git checkout -b response`
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+`git diff`
```

Comment:
We were looking for `git status` which will show you what files have been changed, `git diff` is close but not exactly what we were looking for.